src/Adapters/Cache/InMemoryCache.js

```diff
@@ -0,0 +1,70 @@
+const DEFAULT_CACHE_TTL = 60 * 1000;
```

Comment:
You can create a InMemoryCache adapter with any TTL you want when you initialize the server.

``new ParseServer({cacheAdapter: new InMemoryCache({ttl: 24 * 60 * 1000}) })``

or you can completly disable it by having a cacheAdapter that just ``Promise.resolve(null)`` for gets/sets/del/clear